src/components/table/OcTableFiles.spec.js

```diff
@@ -168,4 +171,19 @@ describe("OcTableFiles", () => {
 
     expect(wrapper.emitted().fileClick[0][0].name).toMatch("forest.jpg")
   })
+
+  it("emitting contextmenuClicked event on right click on table row", async () => {
+    const tableRow = await wrapper.find(".oc-tbody-tr-forest")
```

Comment:
This test is not triggering a right click, so it doesn't match the description. I'd say either remove it (less coverage) or really trigger a right click.